packages/graphql-tool-utilities/config.ts

```diff
@@ -37,9 +36,9 @@ export async function getGraphQLProjectIncludedFilePaths(
 ) {
   return (await Promise.all(
     projectConfig.includes.map((include) =>
-      promisify(glob)(getGraphQLFilePath(projectConfig, include), {
+      promisify(glob)(projectConfig.resolveConfigPath(include), {
         ignore: projectConfig.excludes.map((exclude) =>
-          getGraphQLFilePath(projectConfig, exclude),
+          projectConfig.resolveConfigPath(exclude),
```

Comment:
Hmm this name seems misleading since it does not resolve the config path, it resolves the path relative to the config, no?

Reply:
it's true, the `graphql-config` function name is a bit misleading. but short of adding an augmentation to fix the misleading function name there isn't much to be done. I'm hoping that this is to be rarely used outside of this tooling. That said, anyone using `graphql-config` will have access to `resolveConfigPath` so it's probably not worth fighting too hard. I'll get a PR up in `graphql-config` to refactor the name.

Reply:
I posted a PR but I'm not sure what kind of turn around time to expect.

Reply:
Can we not just add this as a separate method on the projects we return? Since we wrap all the graphql config stuff seems like that should be doable.